unittests/APITests/ReentrantMutex.cpp

```diff
@@ -0,0 +1,154 @@
+#include <catch2/catch.hpp>
+#include <chrono>
+#include <csetjmp>
+#include <FEXCore/Utils/ReentrantMutex.h>
+#include <thread>
+#include <signal.h>
+
+// Test that ensures the Reentrant mutex will timeout without signaling
+TEST_CASE("SimpleWait", "Reentrant") {
```

Comment:
Is this way of specifying the second `TEST_CASE` parameter documented anywhere? The reference docs seem to indicate this is reserved for tags, so it should be "[Reentrant]" instead.

Reply:
Resolved.